dvpOAuth/__init__.py,sha256=yqoW3N0xrTyNRXQUn2A2Xx14flEDoIatIt_KOY1EZBk,162
dvpOAuth/class_OAuth.py,sha256=AFr8nD8idaq3y49Xzv8DecXYGuhnxoGNg90Y8BtmYy8,10855
dvpOAuth/error.py,sha256=Y3BjsevsSGMDVVdn6ztiQMcGNtB6lySkNPoL7y4BT-Q,131
dvpOAuth/util.py,sha256=y-S4zD5BGEWKHVfJKGyy5teIeV2XdWrMXeRubAW54Ek,685
dvpOAuth-0.0.5.dist-info/LICENSE,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
dvpOAuth-0.0.5.dist-info/METADATA,sha256=6IIVEiQdNeZxhc4NKTCtZitLf7WSsMFaUS8z_GkGB3w,991
dvpOAuth-0.0.5.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
dvpOAuth-0.0.5.dist-info/top_level.txt,sha256=14UgmlstFHh0UvlWIljRlFO26u2OesXzFC8SsWFZRmw,9
dvpOAuth-0.0.5.dist-info/RECORD,,
